purchase_minimum_amount/tests/test_po_minimum_amount_block.py

```diff
@@ -122,5 +122,5 @@ def test_po_amount_block_2(self):
             purchase1.approval_block_id,
             self.env['purchase.approval.block.reason'])
 
-        purchase1.sudo(self.user1_id).button_confirm()
+        purchase1.sudo().button_confirm()
```

Comment:
Why not executing as `user1_id` ?

Reply:
@jarroyomorales here Travis was failed because some access right error

Reply:
It is good to use user1_id. The test was probably made to test that user1 could confirm the issue. Please, add it again and check which permissions are missing for the user1